grails-app/services/streama/SettingsService.groovy

```diff
@@ -128,4 +132,20 @@ class SettingsService {
       resultValue.message = "Invalid API language: The entered language is not an IETF language tag.";
     }
   }
+
+  def validateCredentialsForOpensubtitles(Settings settingsInstance, resultValue) {
+    def credentials = settingsInstance.value
+    def url = opensubtitlesService.buildUrl(null, "forrest gump", null, "eng")
+    def response = opensubtitlesService.sendRequest(url, credentials)
+    if (response.statusCodeValue == 403) {
+      resultValue.success = true
+      resultValue.message = "Invalid credentials for opensubtitles: Username or password is incorrect."
```

Comment:
Hm, I am getting this error locally, even though I can login fine using these credentials. If I used the wrong Login, Maybe add instructions to the settings description on where to get these? You can add any kind of link to the description, as well, it is output as html

Reply:
Maybe lets take a look together?